.travis.yml

```diff
@@ -6,7 +6,17 @@ php:
   - 5.6
   - hhvm
 
+env:
+  - PAYLOAD=true
+  - PAYLOAD=false
+
+matrix:
+  exclude:
+    - php: 5.4
+      env: PAYLOAD=true
+
 before_script:
-  - composer install --prefer-source -n
+  - composer install --prefer-source -n --no-dev
+  - '[[ ${PAYLOAD} == "true" ]] && composer require spomky-labs/jose:2.0.x-dev'
```

Comment:
Otherwise, if the PAYLOAD env variable isn't "true", the command will fail and thus the Travis build will fail.